artemis-server/pom.xml

```diff
@@ -156,6 +156,10 @@
          <groupId>io.micrometer</groupId>
          <artifactId>micrometer-core</artifactId>
       </dependency>
+      <dependency>
+         <groupId>org.apache.activemq</groupId>
+         <artifactId>artemis-boot</artifactId>
+      </dependency>
```

Comment:
I don't mean to sound like a broken record, but this seems inverted to me like the issue with `artemis-commons` before. The `artemis-boot` module starts the broker. It doesn't seem like the broker should depend up on it.

Have you considered moving the constant to `artemis-commons` and then making `artemis-boot` depend on `artemis-commons` but excluding all the `artemis-commons` dependencies so you don't get a bunch of transitive stuff? Then the boot classpath can safely be `artemis-boot` and `artemis-commons`.

I also still think it would make sense just to ditch the constant altogether and use the literal instead.

Reply:
boot sets up and environment, and the broker makes use of that, it is communicating via system properties and the constant captures that.  I don't want that env to change without reference to the shared usage. boot is intentionally very small with no deps, it is a good thing to depend on.

Reply:
I think your original argument for not putting it is commons made sense, boot is a better place.

Reply:
Artemis-boot should be the loader.. nothing having a dependency towards boots.


We risk of circular dependencies I guess.